tools/StaticAnalysis/Exceptions/AzureRM.Automation/SignatureIssues.csv

```diff
@@ -59,3 +59,42 @@
+"C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Automation\Microsoft.Azure.Commands.ResourceManager.Automation.dll","Microsoft.Azure.Commands.Automation.Cmdlet.UpdateManagement.NewAzureAutomationSoftwareUpdateConfiguration","New-AzureRmAutomationSoftwareUpdateConfiguration","1","8410","Parameter IncludedPackageNameMasks of cmdlet New-AzureRmAutomationSoftwareUpdateConfiguration does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
+"C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Automation\Microsoft.Azure.Commands.ResourceManager.Automation.dll","Microsoft.Azure.Commands.Automation.Cmdlet.UpdateManagement.NewAzureAutomationSoftwareUpdateConfiguration","New-AzureRmAutomationSoftwareUpdateConfiguration","1","8420","Parameter set '__AllParameterSets' of cmdlet 'New-AzureRmAutomationSoftwareUpdateConfiguration' contains at least one parameter with a position larger than four, which is discouraged.","Limit the number of positional parameters in a single parameter set to four or fewer."
+"C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Automation\Microsoft.Azure.Commands.ResourceManager.Automation.dll","Microsoft.Azure.Commands.Automation.Cmdlet.UpdateManagement.NewAzureAutomationSoftwareUpdateConfiguration","New-AzureRmAutomationSoftwareUpdateConfiguration","1","8420","Parameter set 'Windows' of cmdlet 'New-AzureRmAutomationSoftwareUpdateConfiguration' contains at least one parameter with a position larger than four, which is discouraged.","Limit the number of positional parameters in a single parameter set to four or fewer."
+"C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Automation\Microsoft.Azure.Commands.ResourceManager.Automation.dll","Microsoft.Azure.Commands.Automation.Cmdlet.UpdateManagement.NewAzureAutomationSoftwareUpdateConfiguration","New-AzureRmAutomationSoftwareUpdateConfiguration","1","8420","Parameter set 'Linux' of cmdlet 'New-AzureRmAutomationSoftwareUpdateConfiguration' contains at least one parameter with a position larger than four, which is discouraged.","Limit the number of positional parameters in a single parameter set to four or fewer."
+"C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Automation\Microsoft.Azure.Commands.ResourceManager.Automation.dll","Microsoft.Azure.Commands.Automation.Cmdlet.UpdateManagement.NewAzureAutomationSoftwareUpdateConfiguration","New-AzureRmAutomationSoftwareUpdateConfiguration","1","8510","Cmdlet 'New-AzureRmAutomationSoftwareUpdateConfiguration' has multiple parameter sets, but no defined default parameter set.","Define a default parameter set in the cmdlet attribute."
```

Comment:
Should be removed when DefaultParameterSetName is added.

Reply:
Fixed.